### PR TITLE
Highlight dependencies on hover and focus

### PR DESCRIPTION
**Pull request by goto-bus-stop**
image

When you hover over a tech tree node, it will highlight its dependencies with a white line, in a similar way to the ingame tech tree.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -153,3 +153,13 @@
         .xRef__badge {
             cursor: pointer;
         }
+
+        .connection {
+            fill: none;
+            stroke: #000;
+            stroke-width: 2px;
+        }
+        .connection.is-highlight {
```
Comment (HSZemi) [inline style.css]: I'd set `stroke-width: 3px;` for `.is-highlight`, what do you think?

File: index.html [hunk 2]
```diff
@@ -164,8 +163,10 @@ <h2 id="civname">Loading…</h2>
                         .mouseover(function () {
                             highlightPath(caret.id);
                         })
+                        .mouseout(resetHighlightPath)
                         .click(function () {
                             displayHelp(caret.id);
+                            resetHighlightPath(caret.id);
```
Comment (HSZemi) [inline index.html]: This looks like a parameter to a parameterless function :thinking:

**Comment by goto-bus-stop** [comment]
Thanks for the comments! I didn't think I'd like having multiple paths highlighted at once, but it's actually not bad:

image

**Comment by HSZemi** [comment]
Marvellous, thank you :yellow_heart:

**Pull request merged**